camel-core/src/main/java/org/apache/camel/support/ReloadStrategySupport.java

```diff
@@ -181,6 +181,16 @@ public int getReloadCounter() {
     public int getFailedCounter() {
         return failed;
     }
+    
+    @ManagedOperation(description = "Set succeeded count on xml reload in case overriden")
```

Comment:
I dont think these should be JMX write operations, eg remove the annotation. There can still be a setter for extensions